tests/Functional/Action/Assignment/ListUserAssignmentsActionTest.php

```diff
@@ -126,22 +136,32 @@ public function testItReturnListOfUserAssignmentsEvenWhenCurrentDateDoesNotMatch
         $endDate = $lineItem->getEndAt();
 
         self::assertSame(Response::HTTP_OK, $this->kernelBrowser->getResponse()->getStatusCode());
-        self::assertSame([
-            'assignments' => [
-                [
-                    'id' => $user->getLastAssignment()->getId(),
-                    'username' => $user->getUsername(),
-                    'state' => Assignment::STATE_READY,
-                    'attemptsCount' => $user->getLastAssignment()->getAttemptsCount(),
-                    'lineItem' => [
-                        'uri' => $lineItem->getUri(),
-                        'label' => $lineItem->getLabel(),
-                        'startDateTime' => $startDate instanceof DateTimeInterface ? $startDate->getTimestamp() : '',
-                        'endDateTime' => $endDate instanceof DateTimeInterface ? $endDate->getTimestamp() : '',
-                        'maxAttempts' => $lineItem->getMaxAttempts(),
+        self::assertSame(
+            [
+                'assignments' => [
+                    [
+                        'id' => $user->getLastAssignment()->getId(),
+                        'username' => $user->getUsername(),
+                        'state' => Assignment::STATE_READY,
+                        'attemptsCount' => $user->getLastAssignment()->getAttemptsCount(),
+                        'lineItem' => [
+                            'uri' => $lineItem->getUri(),
+                            'label' => $lineItem->getLabel(),
+                            'isActive' => $lineItem->isActive(),
+                            'startDateTime' => $startDate instanceof DateTimeInterface ? $startDate->getTimestamp(
+                            ) : '',
+                            'endDateTime' => $endDate instanceof DateTimeInterface ? $endDate->getTimestamp() : '',
+                            'maxAttempts' => $lineItem->getMaxAttempts(),
+                        ],
                     ],
                 ],
             ],
-        ], json_decode($this->kernelBrowser->getResponse()->getContent(), true, 512, JSON_THROW_ON_ERROR));
```

Comment:
This was fitting on the line size, why change it?

Reply:
https://github.com/oat-sa/simple-roster/pull/178#discussion_r543309353